code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -36,8 +36,8 @@
 	item_state = "trashbag"
 
 	w_class = 4
-	max_w_class = 2
-	max_storage_space = 56
+	max_w_class = 3

```

Comment:
There are quite a few weapons and other storage containers that have a weight class of 3
This change would allow the trashbag to be used as a ghetto-bag-of-holding, which is not wanted.